sentry-core/src/performance.rs

```diff
@@ -476,6 +476,8 @@ impl TransactionOrSpan {
     }
 
     /// Returns the headers needed for distributed tracing.
+    /// [`crate::Scope::iter_trace_propagation_headers`] is preferred if the intention is to get
+    /// the distributed tracing headers for the currently active span/transaction.
```

Comment:
[nit] try to make this comment a bit more concise:

```suggestion
    /// Use [`crate::Scope::iter_trace_propagation_headers`] to obtain the active 
    /// span's/transaction's distributed tracing headers.
```